.travis.yml

```diff
@@ -1,13 +1,20 @@
 env:
   - NODE_ENV="production"
 language: node_js
-node_js:
-  - "12"
-  - "14"
 services:
   - docker
 before_install:
   # Create a master branch for commitlint
   # https://github.com/conventional-changelog/commitlint/issues/6
   - git remote set-branches --add origin master && git fetch
 install: NODE_ENV=development npm ci
+jobs:
+  include:
+    - stage: Node 12 tests
+      node_js:
+        - "12"
+      script: npm test
```

Comment:
This is the script `./scripts/run-integration-tests.travis.sh` only runs when master is merged